Libraries/StyleSheet/processColor.js

```diff
@@ -13,9 +13,10 @@
 const Platform = require('../Utilities/Platform');
 
 const normalizeColor = require('./normalizeColor');
+import type {ColorValue, ProcessedColorValue} from './ColorValueTypes';
 
 /* eslint no-bitwise: 0 */
-function processColor(color?: ?(string | number)): ?number {
```

Comment:
This file has jest tests, can you add some new tests for this behavior at processColor-test.js?